### PR TITLE
Add imaxdiv, wcscoll, getdtablesize

### PR DESCRIPTION
**Pull request by izissise**
Some added function implementation that are missing when compiling bash

### CODE REVIEW

**Comment by izissise** [comment]
Fixed for contributing guidelines

File: test/libc/fmt/imaxdiv_test.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  res = imaxdiv(0, -11);
+  EXPECT_EQ(0, res.quot);
+  EXPECT_EQ(0, res.rem);
+
+  res = imaxdiv(0, 11);
+  EXPECT_EQ(0, res.quot);
+  EXPECT_EQ(0, res.rem);
+
+  res = imaxdiv(INT64_MIN, -1);
```
Comment (jart) [inline test/libc/fmt/imaxdiv_test.c:54]: 1. `INTMAX_MAX`
2. Why isn't this raising an arithmetic exception? It should. 

```c
int main(int argc, char *argv[]) {
  volatile intmax_t i, j;
  i = INTMAX_MIN;
  j = -1;
  i = i / j;
  return 0;
}
```

**Review by jart — CHANGES_REQUESTED** [review]
Thanks for sending this. Some review comments.

File: test/libc/fmt/imaxdiv_test.c (hunk 1 above)
Comment (jart) [reply test/libc/fmt/imaxdiv_test.c:54]: Scratch that. According to GitHub Actions, it does raise SIGFPE. Good.

Comment (izissise) [reply test/libc/fmt/imaxdiv_test.c]: Oops I meant not to overflow, I fixed the test

**Review by jart — APPROVED** [review]
Looks good. Thank you so much!

**Pull request merged**